<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 253, 174);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 247, 201);
color: rgb(202, 8, 255);</string>
    </property>
    <property name="text">
     <string>Форма решения задач</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endButton">
    <property name="geometry">
     <rect>
      <x>604</x>
      <y>502</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
gridline-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Завершить работу</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="qwestionText">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>661</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ответ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="answerEdit">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
gridline-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Сохранить ответ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previewButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>422</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
gridline-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Предыдущий вопрос</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
gridline-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>Следующий вопрос</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
gridline-color: rgb(170, 0, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;--- Вернуться и бросить всё</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
